src/main/java/com/docusign/esign/client/ApiClient.java

```diff
@@ -1345,7 +1345,7 @@ public File prepareDownloadFile(Response response) throws IOException {
       }
     }
 
-    Entity<?> entity = (body == null) ? Entity.json("") : serialize(body, formParams, contentType);
+    Entity<?> entity = (body == null) ? Entity.json("{}") : serialize(body, formParams, contentType);
```

Comment:
is this intended?

Reply:
Yes and I was about to update the Changelog with the ticket number but still waiting for the ticket management system to be back online...

Reply:
done!

Reply:
👍